mlbaklava/resources/deploy/main.py

```diff
@@ -69,7 +70,7 @@ def application(func):
     else:
         spec = inspect.getfullargspec(func)
 
-    if len(spec.args) != 1:
+    if len(spec.args) != 2:
```

Comment:
What is the reason for changing this `2`?

Reply:
We have validation to have predict exactly of 1 argument, it should be lambda like function

Reply:
There is no point add extra argument to predict method because there is no way to pass extra argument to web container. Everything should be part of payload.